.github/workflows/verify_bundled_wheels.yml

```diff
@@ -0,0 +1,18 @@
+name: Verify bundled pip and setuptools
```

Comment:
Minor suggestion: I know the only other multi-word yml filename in https://github.com/python/cpython/tree/main/.github/workflows uses an underscore:

```console
$ ls .github/workflows
build.yml         build_msi.yml     doc.yml           posix-deps-apt.sh stale.yml
```

But I'd probably go for dashes here, like `posix-deps-apt.sh`.

Reply:
Thank you for the review!

> But I'd probably go for dashes here, like `posix-deps-apt.sh`.

I renamed the file.